ticdc/ticdc-changefeed-config.md

```diff
@@ -36,345 +36,573 @@ Info: {"upstream_id":7178706266519722477,"namespace":"default","id":"simple-repl
+
+- Specifies the duration for which the changefeed is allowed to automatically retry when internal errors or exceptions occur.
+- The changefeed enters the failed state if internal errors or exceptions occur in the changefeed and persist longer than the duration set by this parameter.
+- When the changefeed is in the failed state, you need to restart the changefeed manually for recovery.
+- The format is `"h m s"`. For example, `"1h30m30s"`.
+- Default value: `"30m"`
+
+### mounter
+
+#### `worker-num`
+
+- Specifies the number of threads with which the mounter decodes KV data.
+- Default value: `16`
+
+### filter
+
+#### `ignore-txn-start-ts`
+
+- Ignores the transaction of specified start_ts.
+
+<!-- Example: `[1, 2]` -->
+
+#### `rules`
+
+- Specifies the filter rules. For more information, see [Syntax](/table-filter.md#syntax).
+
+<!-- Example: `['*.*', '!test.*']` -->
+
+#### filter.event-filters
+
+<!-- The detailed syntax is described in https://docs.pingcap.com/tidb/stable/ticdc-filter -->
+
+##### `matcher`
+
+- `matcher` is an allow list. `matcher = ["test.worker"]` means this rule only applies to the `worker` table in the `test` database.
+
+##### `ignore-event`
+
+- `ignore-event = ["insert"]` ignores `INSERT` events. 
+- `ignore-event = ["drop table", "delete"]` ignores the `DROP TABLE` DDL events and the `DELETE` DML events. Note that when a value in the clustered index column is updated in TiDB, TiCDC splits an `UPDATE` event into `DELETE` and `INSERT` events. TiCDC cannot identify such events as `UPDATE` events and thus cannot correctly filter out such events.
+
+##### `ignore-sql`
+
+- `ignore-sql = ["^drop", "add column"]` ignores DDLs that start with `DROP` or contain `ADD COLUMN`.
+
+##### `ignore-delete-value-expr`
+
+- `ignore-delete-value-expr = "name = 'john'"` ignores `DELETE` DMLs that contain the condition `name = 'john'`.
+
+##### `ignore-insert-value-expr`
+
+- `ignore-insert-value-expr = "id >= 100"` ignores `INSERT` DMLs that contain the condition `id >= 100`。
+
+##### `ignore-update-old-value-expr`
+
+- `ignore-update-old-value-expr = "age < 18"` ignores `UPDATE` DMLs whose old value contains `age < 18`。
+
+##### `ignore-update-new-value-expr`
+
+- `ignore-update-new-value-expr = "gender = 'male'"` ignores `UPDATE` DMLs whose new value contains `gender = 'male'`.
+
+### scheduler
+
+#### `enable-table-across-nodes`
+
+- Allocate tables to multiple TiCDC nodes for replication on a per-Region basis.
+- This configuration item only takes effect on Kafka changefeeds and is not supported on MySQL changefeeds.
+- When `enable-table-across-nodes` is enabled, there are two allocation modes:
+
+    1. Allocate tables based on the number of Regions, so that each TiCDC node handles roughly the same number of Regions. If the number of Regions for a table exceeds the value of `region-threshold`, the table will be allocated to multiple nodes for replication. The default value of `region-threshold` is `10000`.
```

Comment:
```suggestion
    1. Allocate tables based on the number of Regions, so that each TiCDC node handles roughly the same number of Regions. If the number of Regions for a table exceeds the value of [`region-threshold`](#region-threshold), the table will be allocated to multiple nodes for replication. The default value of `region-threshold` is `10000`.
```